src/temp/Explore/Explore.module.scss

```diff
@@ -1,6 +1,7 @@
+@import '../../theme/theme';
```

Comment:
`The Sass team discourages the continued use of the @import rule. Sass will [gradually phase it out](https://github.com/sass/sass/blob/main/accepted/module-system.md#timeline) over the next few years, and eventually remove it from the language entirely. Prefer the [@use rule](https://sass-lang.com/documentation/at-rules/use) instead.`
Replace this with @use instead. Use alias `theme` to keep things common